packages/@aws-cdk/cfnspec/spec-source/500_RDS_DBProxyTargetGroup_TargetGroupName_patch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Running a simple test, it looks like the patching operation does not fail if the property already exists and simply overwrites.

In this case, I would prefer to leave the code as it is and not move it into cfnspec. When CloudFormation does correct its spec, I would rather have its original part of its spec rather than our  patch.

On the other hand, if we can find a way to fail the build of `cfnspec` when CloudFormation ends up adding this property into its spec as a way to remove this, then that would be ok.